docs/src/man/query_package.md

```diff
@@ -0,0 +1,69 @@
+# Query package
+
+The [Query.jl](https://github.com/davidanthoff/Query.jl) package provides advanced data manipulation capabilities for `DataFrames` (and many other data structures). This section provides a short introduction to the package, the [Query.jl](https://github.com/davidanthoff/Query.jl) [documentation](http://www.david-anthoff.com/Query.jl/stable/) has a more comprehensive documentation of the package.
+
+To get started, install the Query package:
+
+```julia
+Pkg.add("Query")
+```
+
+A query is started with the `@from` macro and consists of a series of query commands. Query provides commands that can filter, project, join, group, flatten and group data from a `DataFrame`. A query returns either an iterator of the transformed data, or one can materialize the results of a query into a variety of data structures, including a new `DataFrame`.
```

Comment:
Query -> Query.jl? Else it's going to be quite confusing with the term "query".

Syntax is a bit weird in the second part of the sentence: it doesn't match well with "returns either".